pkg/flowkit/state.go

```diff
@@ -107,14 +106,28 @@ func (p *State) SaveEdited(paths []string) error {
 		}
 	}
 
-	return p.Save(paths[0], p.accountLocations)
+	return p.Save(paths[0])
 }
 
 // Save saves the project configuration to the given path.
-func (p *State) Save(path string, accountLocations map[string]string) error {
-	p.conf.Accounts = accountsToConfig(*p.accounts, accountLocations)
+func (p *State) Save(path string) error {
+	p.conf.Accounts = accountsToConfig(*p.accounts, p.confLoader.AccountsFromFile())
 	err := p.confLoader.Save(p.conf, path)
 
+	// if we have defined accounts to be saved to private file iterate over them and save them separately
```

Comment:
```suggestion
	// if we have defined accounts to be saved to an external file, iterate over them and save them separately
```

This feature isn't just for private files, so don't want to confuse future maintainers